packages/amplify_datastore/android/src/test/kotlin/com/amazonaws/amplify/amplify_datastore/AmplifyDataStorePluginTest.kt

```diff
@@ -156,45 +159,113 @@ class AmplifyDataStorePluginTest {
         doAnswer { invocation: InvocationOnMock ->
             (invocation.arguments[1] as Consumer<DataStoreItemChange<SerializedModel>>).accept(
                     dataStoreItemChange)
-            null as Void?
+            null
         }.`when`(mockAmplifyDataStorePlugin).delete(any(),
-                ArgumentMatchers.any<
-                        Consumer<DataStoreItemChange<SerializedModel>>>(),
-                ArgumentMatchers.any<Consumer<DataStoreException>>())
+                                                    any<Consumer<DataStoreItemChange<SerializedModel>>>(),
+                                                    any<Consumer<DataStoreException>>())
 
         flutterPlugin.onDelete(mockResult,
-                readMapFromFile("delete_api",
-                        "request/instance_no_predicate.json",
-                        HashMap::class.java) as HashMap<String, Any>)
+                               readMapFromFile("delete_api",
+                                               "request/instance_no_predicate.json",
+                                               HashMap::class.java) as HashMap<String, Any>)
 
         verify(mockResult, times(1)).success(null)
     }
 
     @Test
-    fun test_Delete_Error() {
+    fun test_delete_error() {
 
-        var dataStoreException = DataStoreException("AmplifyException", DataStoreException.REPORT_BUG_TO_AWS_SUGGESTION)
+        var dataStoreException = DataStoreException("AmplifyException",
+                                                    DataStoreException.REPORT_BUG_TO_AWS_SUGGESTION)
 
         doAnswer { invocation: InvocationOnMock ->
-                (invocation.arguments[2] as Consumer<DataStoreException>).accept(
-                        dataStoreException)
-                null as Void?
+            (invocation.arguments[2] as Consumer<DataStoreException>).accept(
+                    dataStoreException)
+            null
         }.`when`(mockAmplifyDataStorePlugin).delete(any(),
-                ArgumentMatchers.any<
-                        Consumer<DataStoreItemChange<SerializedModel>>>(),
-                ArgumentMatchers.any<Consumer<DataStoreException>>())
+                                                    any<Consumer<DataStoreItemChange<SerializedModel>>>(),
+                                                    any<Consumer<DataStoreException>>())
 
         flutterPlugin.onDelete(mockResult,
-                readMapFromFile("delete_api",
-                        "request/instance_no_predicate.json",
-                        HashMap::class.java) as HashMap<String, Any>)
+                               readMapFromFile("delete_api",
```

Comment:
Overall I think the whitespace changes in this file are sort of moving in the wrong direction. +4 or +8 space indents would be alright. Looks like maybe these started as a couple of tabs, but are being rendered as k * 8 many spaces, which gets a little wild.

Reply:
my android studio is set to align functional call arguments for better readability. Is this not desirable?